spec/classes/init_spec.rb

```diff
@@ -232,6 +236,19 @@
           it { is_expected.to create_logrotate__rule('simp_rsyslog_server_profile').with_lastaction_restart_logger(true) }
         end
 
+        context 'simp_rsyslog class that has moved the logdir' do
+          let(:params) {{
+            :is_server => true,
+          }}
+          let(:hieradata) { 'rsyslog_server_logdir' }
+          it_behaves_like 'a structured module'
+          it { is_expected.to contain_rsyslog__rule__local('10_00_default_cron').with({
+            'rule'            => "prifilt('cron.*')",
+            'dyna_file'       => '/opt/logs/%DOMAIN%/cron.log',
```

Comment:
Can we change this to something that's actually valid?

We recommend that users take a look at the acceptance tests when they don't understand something and this won't actually work. I think it will put everything in a directory called `%DOMAIN%`.

Reply:
Yeah, I made that up. Looking for a real one now.